src/org/jitsi/util/TimeUtils.java

```diff
@@ -179,15 +179,11 @@ public static long getLsw(long ntpTime)
     /**
      * Returns the difference between two RTP timestamps.
      * @return the difference between two RTP timestamps.
+     * DEPRECATED: use RTPUtils.timeStampDiff
```

Comment:
Typo. Perhaps we ca use "@deprecated use {@link RTPUtils#rtpTimestampDiff(long, long)}"?

Can we also update the now deprecated use of TimeUtils#rtpDiff within libjitsi?